pkg/operator/hive/hive.go

```diff
@@ -333,3 +373,27 @@ func (r *ReconcileHiveConfig) runningOnOpenShift(hLog log.FieldLogger) (bool, er
 
 	return len(list.APIResources) > 0, nil
 }
+
+func computeHiveControllersConfigHash(data map[string]string) string {
+	hasher := md5.New()
+	hasher.Write([]byte(fmt.Sprintf("%v", data)))
+	return hex.EncodeToString(hasher.Sum(nil))
+}
+
+func setHiveControllersConfig(config *hivev1.ControllerConfig, hiveControllersConfigMap *corev1.ConfigMap, controllerName string) {
```

Comment:
Let's be consistent between `computeHiveControllersConfigHash` and `setHiveControllersConfig` with whether we are passing in just the data map or the full ConfigMap.